### PR TITLE
Add release notes for 0.105

### PR DESCRIPTION
**Pull request by nileema**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.105.rst [hunk 1]
```diff
@@ -11,3 +11,11 @@ SPI Changes
     This is a backwards incompatible change with the previous connector SPI.
     If you have written a connector, you will need to update your code
     before deploying this release.
+
+General Changes
+---------------
+
+* Fix page removal in ``SharedBuffer`` which can cause queries to be permanently blocked.
+* Fix ``ConcurrentModificationException`` in ``ClusterMemoryManager``.
```
Comment (cberner) [inline presto-docs/src/main/sphinx/release/release-0.105.rst]: You can leave this one out. I don't think it's noteworthy

**Comment by cberner** [comment]
looks good

File: presto-docs/src/main/sphinx/release/release-0.105.rst [hunk 2]
```diff
@@ -11,3 +11,13 @@ SPI Changes
     This is a backwards incompatible change with the previous connector SPI.
     If you have written a connector, you will need to update your code
     before deploying this release.
+
+General Changes
+---------------
+
+* Fix page removal in ``SharedBuffer`` which can cause queries to be permanently blocked.
+* Make ``JdbcRecordCursor.close()`` idempotent.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.105.rst]: Close connections correctly in JDBC connectors.

File: presto-docs/src/main/sphinx/release/release-0.105.rst [hunk 3]
```diff
@@ -11,3 +11,13 @@ SPI Changes
...
+* Close connection correctly in ``JdbcRecordCursor``.
+* Add implicit coercions for values of equijoin criteria.
+* Recursively check presence of ``OVER`` clause in window functions
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.105.rst]: Fix detection of nested window functions.

**Pull request merged**